lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -290,7 +283,6 @@
      * @param {String} key the specific key we are looking for in the extension, such as "co"
      */
     const UnicodeExtensionValue = function (extension, key) {
-        raise.assert(key.length === 2);
```

Comment:
>raise.assert(key.length === 2); [](start = 8, length = 31)

was this assert not valid?

Reply:
meh -- the spec has lots of lines like "assert x equals some condition." this was intended to replicate that, but its not clear what value it has (its not clear to me what error we should throw on such an assert or if we should failfast, and we already have numerous failfasts in the internal c++ bits.